### ISSUE TITLE
Several issues after updating to Windows 11 Build 230624-1828

### ISSUE THREAD

**Issue by sitistas** [Erisa/RoundedTB#308]
After the last update of Windows 11 (Build 230624-1828), that includes the Windows copilot and other new features, the behavior of the taskbar seems to have changed. something that messes up the **dynamic mode**. My setup includes two monitors.

When you I select the **dynamic mode**, the behavior of the dynamic mode is completely random. 

## 1. When the option "Fill taskbar when maximised" is enabled

In the first screenshot, my active window is not at that monitor, and many icons disappear. 
![image](https://github.com/Erisa/RoundedTB/assets/81265818/6408c515-f1a5-4692-af5b-ab24f0aac4cf)

If I move my active window to that specific monitor, then the dynamic mode seems to work correctly. However, if there is no other maximised window in the other monitor, the same thing as above happens to my other monitor.
![image](https://github.com/Erisa/RoundedTB/assets/81265818/3aeab50e-93a9-430e-8243-1d861f88bc8f)

## 2.  When the option "Fill taskbar when maximised" is disabled

This makes the situation even worse, and no matter what I try, I just see one or two icons from the taskbar, that seems to be in the center of the screen. This applies to all the monitors.
![image](https://github.com/Erisa/RoundedTB/assets/81265818/cb86fc37-e82c-400e-bb38-f6504496ec37)
![image](https://github.com/Erisa/RoundedTB/assets/81265818/f4bb85fe-065a-4500-af61-c5a936dc8433)

For the moment, I will stick to the non-dynamic mode to avoid these issues, and I will try to add more comments if I find any more issues.

**Comment by anoopadvaitha** [comment]
I'm having same issue too

**Comment by alecames** [comment]
Having same issue, hides a lot of icons on dynamic:

Normal
![image](https://github.com/RoundedTB/RoundedTB/assets/18179415/40e77263-813a-4149-82da-b9dbf2fa3c0c)

Dynamic
![image](https://github.com/RoundedTB/RoundedTB/assets/18179415/7de73cc1-7e1b-4894-b77a-387334a7dc05)

**Comment by computertester10** [comment]
I have the same problem on the dev channel.

Normal:
![image](https://github.com/RoundedTB/RoundedTB/assets/121703698/bc272243-0f37-4e1e-9aa6-eb65e0ee1113)

Dynamic:
![image](https://github.com/RoundedTB/RoundedTB/assets/121703698/8c843b61-4dab-4617-ae3f-1faeb16f33d4)

**Comment by wrecklass** [comment]
Agreed, this has become a problem with the latest updates to Windows 11. I found a workaround: Set RoundedTB Inner Margin to -100. That may change depending on how many icons people have on their taskbar. Hopefully this will help uncover the fix.

![Screenshot 2023-08-20 142413](https://github.com/RoundedTB/RoundedTB/assets/663753/71b51258-aa39-45e1-a683-22e2a4907121)

**Comment by computertester10** [comment]
Wow, It actually works, thanks!

**Comment by suaide** [comment]
It's a workaround that works if the number of icons does not increase too much. But it can show some hint on how to fix the problem once for all

**Comment by Erickgtgamas** [comment]
Having it ~ -70 seems to be the best number for the inner margin. But noticed it works well for 1920x1080
![image](https://github.com/RoundedTB/RoundedTB/assets/56701853/e19f3cc5-79ff-4bd5-b641-be061be16a4b)


Photo of how it looks like
![image](https://github.com/RoundedTB/RoundedTB/assets/56701853/a4d5fc71-4855-4b1c-80b2-0ed8952c33d0)

**Comment by computertester10** [comment]
It really just depends on how many icons you have. 👍 :)

**Comment by lothkang** [comment]
I have the same issue on Win11 Pro 23H2 build 22631.2199.
I hope it'll be fixed soon.

**Comment by computertester10** [comment]
One issue though, if you have widgets enabled, they don't show. 🧐
